src/main/java/appeng/core/sync/packets/PacketPatternSlot.java

```diff
@@ -103,18 +142,13 @@ private void writeItem( IAEItemStack slotItem, ByteBuf data ) throws IOException
 		else
 		{
 			data.writeBoolean( true );
-			slotItem.writeToPacket( data );
-		}
-	}
-
-	@Override
-	public void serverPacketData( INetworkInfo manager, AppEngPacket packet, EntityPlayer player )
-	{
-		EntityPlayerMP sender = (EntityPlayerMP) player;
-		if( sender.openContainer instanceof ContainerPatternTerm )
-		{
-			ContainerPatternTerm patternTerminal = (ContainerPatternTerm) sender.openContainer;
-			patternTerminal.craftOrGetItem( this );
+			try
+			{
+				slotItem.writeToPacket( data );
+			}
+			catch( IOException e )
+			{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)